shared/teams/really-leave-team/container-chat.js

```diff
@@ -31,10 +30,11 @@ const mapDispatchToProps = (dispatch: Dispatch, {navigateUp, routeProps}) => ({
 
 export default compose(
   connect(mapStateToProps, mapDispatchToProps),
-  branch(props => props._lastOwner, renderComponent(LastOwnerDialog)),
   lifecycle({
     componentDidMount: function() {
-      this.props._loadTeam(this.props.name)
+      !this.props.loaded && this.props._loadOperations(this.props.name)
     },
-  })
+  }),
+  branch(props => !props.loaded, renderComponent(Spinner)),
```

Comment:
lets not build more of this type of branch code, lets actually make a small component that does the switching